### ISSUE TITLE
Bug Report: Installer doesn't see os-agent

### ISSUE THREAD

**Issue by dautushenka** [home-assistant/supervised-installer#279]
### OS Version

Debian GNU/Linux 11 (bullseye)

### System Information

Linux raspberrypi-2 5.15.84-v8+ #1613 SMP PREEMPT Thu Jan 5 12:03:08 GMT 2023 aarch64 GNU/Linux

### What happened?

Hello everybody!

I have raspberry Pi 4 and fresh installed "Raspberry Pi OS Lite" (Release date: February 21st 2023)
I installed all packages that are required (including os-agent)
The installation script `dpkg -i homeassistant-supervised.deb` doesn't see os-agent
![image](https://user-images.githubusercontent.com/7254705/222830351-098f6b0b-cd7b-4d51-b4ed-cd06393e1ef5.png)
I checked that os-agent is installed and running
![image](https://user-images.githubusercontent.com/7254705/222830409-27973f92-a20a-4b0a-8539-e31e7ee1ba4f.png)
![image](https://user-images.githubusercontent.com/7254705/222830425-39c11367-a2c6-4c8e-b099-0bb59e26f185.png)

So I decided ignore this error and run installation script with `--force-all` argument, it went further, however on "Restart NetworkManager" I had been disconnected from raspberry and could not connect again without hardware reboot(

### Machine Type

raspberrypi4-64

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by ikifar2012** [comment]
Raspberry Pi OS is not supported, only the suppored OS images can be found here https://raspi.debian.net/tested-images/

**Comment by dautushenka** [comment]
It's very suddenly, about a year ago I was able to use this installer with "Raspberry Pi OS". I don't see way to install supported OS images in headless mode, going to play with "Raspberry Pi OS" for a while

**Comment by frenck** [comment]
> It's very suddenly

It never was supported. You might have missed it while reading the requirements.

**Comment by dautushenka** [comment]
> > It's very suddenly
> 
> It never was supported. You might have missed it while reading the requirements.

yep, during reading  the requirements I saw that Debian 11 is good (Raspberry Pi OS also is built on this Debian version), I thought it's enough. Also I was able to use this script before with Raspberry Pi OS, looks like it was luck, currently it's much harder to do it.
